cloudinit/tests/test_util.py

```diff
@@ -730,6 +730,36 @@ def already_mounted_device(self, already_mounted_device_and_mountdict):
         """already_mounted_device_and_mountdict, but return only the device"""
         return already_mounted_device_and_mountdict[0]
 
+    @mock.patch("platform.system")
+    @mock.patch("cloudinit.util.subp.subp")
+    @mock.patch("cloudinit.temp_utils.tempdir")
+    def test_normalize_vfat_on_bsd(self, m_tmpdir, m_subp, m_is_BSD):
+        m_is_BSD.return_value = "SomeBSD"
+        m_tmpdir.__enter__ = mock.Mock(return_value="/tmp/fake")
+        m_tmpdir.__exit__ = mock.Mock(return_value=True)
+        callback = mock.Mock()
+
+        util.mount_cb('/dev/fake0', callback, mtype='vfat')
+        m_subp.assert_called_with(
```

Comment:
If we're using `assert_*` methods then we need `autospec=True` on the corresponding patch call; (search for "autospec" in https://cloudinit.readthedocs.io/en/latest/topics/hacking.html#testing).